CONTRIBUTING.md

```diff
@@ -18,14 +17,21 @@ This documented is primarily aimed at the following repositories:
 - [storage](https://github.com/containers/storage)
 - [libhvee](https://github.com/containers/libhvee)
 - [psgo](https://github.com/containers/psgo)
+- [netavark](https://github.com/containers/netavark)
+- [aardvark](https://github.com/containers/aardvark-dns/)
 
-However most of the things here listed are very generic when contributing to public projects
+This document is not specific to any language; language-specific guidelines are contained in other files.
+Rules specific to the Rust language can be found [here](./CONTRIBUTING_RUST.md).
+Rules specific to the Go language can be found [here](./CONTRIBUTING_GO.md).
```

Comment:
since podman-py is the only python project we have, I believe including a cross ref to https://github.com/containers/podman-py/blob/main/CONTRIBUTING.md is enough. I wouldn't do CONTRIBUTING_PYTHON.md so maybe here?